releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -4985,7 +5630,7 @@
   "125582": {
     "commit": "921b69b133db6b0a919e9eb766ff185277cabc79",
     "text": "kubeadm: added the ControlPlaneKubeletLocalMode feature gate. It can be used to tell kubeadm to use the local kube-apiserver endpoint for the kubelet when creating a cluster with \"kubeadm init\" or when joining control plane nodes with \"kubeadm join\".  The \"kubeadm join\" workflow now includes two new experimental phases called \"control-plane-join-etcd\" and \"kubelet-wait-bootstrap\" which will be used when the feature gate is enabled. This phases will be marked as non-experimental when ControlPlaneKubeletLocalMode becomes GA. During \"kubeadm upgrade\" commands, if the feature gate is enabled, modify the \"/etc/kubernetes/kubelet.conf \" to use the local kube-apiserver endpoint. This upgrade mechanism will be removed once the feature gate goes GA and is hardcoded to true.",
-    "markdown": "Kubeadm: added the ControlPlaneKubeletLocalMode feature gate. It can be used to tell kubeadm to use the local kube-apiserver endpoint for the kubelet when creating a cluster with \"kubeadm init\" or when joining control plane nodes with \"kubeadm join\".  The \"kubeadm join\" workflow now includes two new experimental phases called \"control-plane-join-etcd\" and \"kubelet-wait-bootstrap\" which will be used when the feature gate is enabled. This phases will be marked as non-experimental when ControlPlaneKubeletLocalMode becomes GA. During \"kubeadm upgrade\" commands, if the feature gate is enabled, modify the \"/etc/kubernetes/kubelet.conf \" to use the local kube-apiserver endpoint. This upgrade mechanism will be removed once the feature gate goes GA and is hardcoded to true. ([#125582](https://github.com/kubernetes/kubernetes/pull/125582), [@chrischdi](https://github.com/chrischdi)) [SIG Cluster Lifecycle]",
+    "markdown": "Kubeadm: added the ControlPlaneKubeletLocalMode feature gate. It can be used to tell kubeadm to use the local kube-apiserver endpoint for the kubelet when creating a cluster with \"kubeadm init\" or when joining control plane nodes with \"kubeadm join\".  The \"kubeadm join\" workflow now includes two new experimental phases called \"control-plane-join-etcd\" and \"kubelet-wait-bootstrap\" which will be used when the feature gate is enabled. This phases will be marked as non-experimental when ControlPlaneKubeletLocalMode becomes GA. During \"kubeadm upgrade\" commands, if the feature gate is enabled, modify the \"/etc/kubernetes/kubelet.conf \" to use the local kube-apiserver endpoint. This upgrade mechanism will be removed once the feature gate goes GA and is hardcoded to true. ([#125582](https://github.com/kubernetes/kubernetes/pull/125582), [@chrischdi](https://github.com/chrischdi))",
```

Comment:
```suggestion
    "markdown": "Kubeadm: added the ControlPlaneKubeletLocalMode feature gate. It can be used to tell kubeadm to use the local kube-apiserver endpoint for the kubelet when creating a cluster with \"kubeadm init\" or when joining control plane nodes with \"kubeadm join\".  The \"kubeadm join\" workflow now includes two new experimental phases called \"control-plane-join-etcd\" and \"kubelet-wait-bootstrap\" which will be used when the feature gate is enabled. This phases will be marked as non-experimental when ControlPlaneKubeletLocalMode becomes GA. During \"kubeadm upgrade\" commands, if the feature gate is enabled, modify the \"/etc/kubernetes/kubelet.conf \" to use the local kube-apiserver endpoint. This upgrade mechanism will be removed once the feature gate goes GA and is hardcoded to true. ([#125582](https://github.com/kubernetes/kubernetes/pull/125582), [@chrischdi](https://github.com/chrischdi)) [SIG Cluster Lifecycle]",
```